roles/update/tasks/main.yml

```diff
@@ -44,6 +44,51 @@
     - cifmw_ci_gen_kustomize_values_installplan_approval is defined
     - cifmw_ci_gen_kustomize_values_installplan_approval | lower == 'manual'
 
+- name: Initialize the openstack operator if needed
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit) }}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    definition: "{{ _openstack_init_resource }}"
+    state: present
+  vars:
+    _openstack_init_resource:
+      apiVersion: operator.openstack.org/v1beta1
+      kind: OpenStack
+      metadata:
+        name: openstack
+        namespace: openstack-operators
+  when:
+    - cifmw_ci_gen_kustomize_values_deployment_version is defined
+    - cifmw_ci_gen_kustomize_values_deployment_version is in ['v1.0.3', 'v1.0.6']
```

Comment:
If I understand it right, currently we do support only `v1.0.3` and `v1.0.6` [src](https://github.com/openstack-k8s-operators/ci-framework/blob/58bc72099dfcc8fb9440b3bd8c50e91c5d1b33fd/roles/ci_gen_kustomize_values/README.md) with a `v1.0.7` [like here](https://github.com/openstack-k8s-operators/architecture/blob/6980941078e83c1d6876a2e59981f811f1141b95/examples/common/olm-subscriptions/README.md) (but it currently does not exist).
Is this check required?